winter/schema/docstring_parser.py

```diff
@@ -0,0 +1,21 @@
+import docstring_parser
+
+
+class DocstringParser:
```

Comment:
Consider renaming it to `Docstring` because only constructor does parsing, and the class basically represents already parsed docstring structure.